provider/terraform/website/docs/r/dataproc_cluster.html.markdown

```diff
@@ -239,6 +265,17 @@ The `cluster_config.master_config` block supports:
 	* `disk_config.num_local_ssds` - (Optional) The amount of local SSD disks that will be
 	attached to each master cluster node. Defaults to 0.
 
+* `accelerators` (Optional) The Compute Engine accelerator (GPU) configuration for these instances. Can be specified multiple times.
+
+    * `accelerators.accelerator_type` - (Required) Full URL, partial URI, or short name of the accelerator type resource to expose to this instance.
```

Comment:
Hmm I don't know how I feel about `accelerators.accelerator_type` since that's not actually a way that you'd be able to reference it (maybe just `accelerators.[*].accelerator_type` or something like that)

Reply:
Sure - I was copying other fields in this file, but they were inconsistent anyways. I got rid of all the prefixes.